### PR TITLE
Fix/task eval reload

### PR DESCRIPTION
**Pull request by sukesh02**
### Functionality:
The Drag and drop functionality in Task Evaluation page and Kanban Board

### Solution:
Fixed it using an If statement

### Risk level:
- [x] high 
- [ ] medium
- [ ] low

### How to test:
Now the drag and drop functions without an API error.

### CODE REVIEW

**Review by sanjaykrishna1203 — APPROVED** [review]
Approved

**Review by viv9k — APPROVED** [review]
Approved

File: functions/model/tasks/tark/editTask.js [hunk 1]
```diff
@@ -197,6 +197,10 @@ exports.editTask = function(request, response) {
             promises.push(updateSprintPromise);
         }
 
+        if(milestoneId == undefined) {
+            milestoneId=""
+        }
+
```
Comment (viv9k) [inline functions/model/tasks/tark/editTask.js:203]: The best practice would be to handle this milestone issue from UI, better not to send undefined values to backend.

**Review by viv9k — CHANGES_REQUESTED** [review]
Please check the required comment

**Review by Pendyala-thanvi — APPROVED** [review]

**Review by viv9k — CHANGES_REQUESTED** [review]
This might not be a good idea to send milestone as empty string as some tasks might be attached to milestone and therefore If they update anything into the task from task evaluation and kanban board the milestone will be updated to empty string, not a good idea.

Milestone should be attached to previous value of milestone and if there is no previous value for milestone only then send an empty string value.

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**